### PR TITLE
Fix: Correction to laser pulse formular for few-cycle wavepackets

### PR DESCRIPTION
**Pull request by stetie**
Similiar to #851, this corrects the laser pulse formular to account for extra terms arrising from deriving the formular from the vector potential. It ensures the integration along the propagation direction is 0. This is important for few-cycle laser pulses.

### CODE REVIEW

**Comment by PrometheusPi** [comment]
@stetie Great job. :+1:

@ax3l Please add @stetie to the developer list.
@BeyondEspresso When we discussed about this for #851, you said this approach only works for constant transversal envelopes. Is this true?

**Comment by ax3l** [comment]
added to developer team, welcome and thx for the first pull request! :rocket: 

just `git commit --amend` to the last commit and `push -f` it to trigger a new CI build :)

**Comment by ax3l** [comment]
P.S.: pls reformat the [style of your commit message](https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/doc/COMMIT.md) a little :)

File: src/picongpu/include/fields/laserProfiles/laserWavepacket.hpp [hunk 1]
```diff
@@ -56,6 +59,9 @@ HINLINE float3_X laserLongitudinal(uint32_t currentStep, float_X& phase)
     const double endUpramp = -0.5 * LASER_NOFOCUS_CONSTANT;
     const double startDownramp = 0.5 * LASER_NOFOCUS_CONSTANT;
 
+    const double tau = PULSE_LENGTH / sqrt(2.0);
```
Comment (ax3l) [inline src/picongpu/include/fields/laserProfiles/laserWavepacket.hpp]: gentle reminder to your spotted correction. same as in plane wave: should be
```c++
const double tau = PULSE_LENGTH / sqrt(2.0);
```

**Comment by PrometheusPi** [comment]
Sinus pulse:
stefanlasersin

sum/amax = 1e-2 is OK

**Comment by PrometheusPi** [comment]
Cosinus pulse:
stefanlasercos

sum/amax = 1.5e-2 is OK

**Comment by ax3l** [comment]
weird offset in the envelope... hm.

**Pull request merged**